src/platform-includes/sourcemaps/overview/javascript.astro.mdx

```diff
@@ -28,36 +28,55 @@ export default defineConfig({
 });
 ```
 
-### Working With Old Authentication Tokens
+### Disable Source Maps Upload
 
-Source maps work best with [organization-scoped auth tokens](/product/accounts/auth-tokens/#organization-auth-tokens). If you are using an old self-hosted Sentry version that doesn't yet support org-based tokens or you're using a different type of Sentry auth token, you'll need to additionally specify your `org` slug in your `sourceMapsUploadOptions`:
+You can disable automatic source maps upload in your Astro config:
 
 ```javascript {filename:astro.config.mjs}
 export default defineConfig({
   integrations: [
     sentry({
       // Other Sentry options
       sourceMapsUploadOptions: {
-        project: "___PROJECT_SLUG___",
-        org: "___ORG_SLUG___",
-        authToken: process.env.SENTRY_AUTH_TOKEN,
+        enabled: false,
       },
     }),
   ],
 });
 ```
 
-### Disable Source Maps Upload
+### Setting Source Maps Assets directory
 
-You can disable automatic source maps upload in your Astro config:
+By default, the Sentry Astro integration will look for source maps in sensible default directories, depending on your `outDir`, `rootDir` and `adapter` configuration.
+If these defaults don't work for you (e.g. due to an advanced customized build setup or an unsupported adapter), you can specify the `assets` option to point to the folder(s) where your source maps are located:
```

Comment:
We don't use i.e. and e.g. according to our style guide.

```suggestion
If these defaults don't work for you (for example, due to an advanced customized build setup or an unsupported adapter), you can specify the `assets` option to point to the folder(s) where your source maps are located:
```

Reply:
right, thx for catching!